compiler/luci/import/src/Importer.cpp

```diff
@@ -122,12 +122,19 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
   }
 
   // Create CircleConst nodes for constant tensors.
+  // Create CircleNodes for constant tensors.
```

Comment:
These two comments conflict.

Reply:
Removed `  // Create CircleConst nodes for constant tensors.` comment!